<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiptWindow</class>
 <widget class="QDialog" name="ReceiptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_main">
     <item>
      <widget class="QScrollArea" name="scrollArea_main">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>338</width>
          <height>538</height>
         </rect>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>221</width>
           <height>261</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_receiptImage">
          <item>
           <widget class="QScrollArea" name="scrollArea_receiptImage">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_receiptImage">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>230</width>
               <height>250</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>250</height>
              </size>
             </property>
             <widget class="QLabel" name="label_receiptImage">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>221</width>
                <height>241</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>216</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QPushButton" name="editCustomersButton">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>20</y>
           <width>80</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Edit
 customers</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>300</y>
           <width>301</width>
           <height>221</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Sitka</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
